Source/WebCore/rendering/EventRegion.cpp

```diff
@@ -208,8 +208,44 @@ bool EventRegionContext::shouldConsolidateInteractionRegion(IntRect bounds, Rend
     return false;
 }
 
+void EventRegionContext::addOcclusionMarginsForSmallInteractionRegions()
+{
+    static int minimumSize = 20;
+    static int occlusionMargin = 10;
+    Vector<InteractionRegion> interactionRegionsWithOcclusions;
+    bool addedOcclutions = false;
```

Comment:
This variable currently seems unnecessary, since you could just check `!interactionRegionsWithOcclusions.isEmpty()` in its place.

(it's also a misspelling as-is)